src/Autofac/Core/Activators/Reflection/ConstructorParameterBinding.cs

```diff
@@ -66,7 +66,7 @@ public class ConstructorParameterBinding
         /// <param name="context">Context in which to construct instance.</param>
         public ConstructorParameterBinding(
             ConstructorInfo ci,
-            IEnumerable<Parameter> availableParameters,
+            Parameter[] availableParameters,
```

Comment:
A breaking change but I don't see this class being called directly so probably fine.